spring/spring-web-reactive/src/test/java/io/quarkus/ts/spring/web/reactive/bootstrap/AbstractSpringWebRestReactiveIT.java

```diff
@@ -23,7 +22,7 @@ public abstract class AbstractSpringWebRestReactiveIT {
 
     @Test
     public void whenGetNotExistBookById_thenNotFound() {
-        final Response response = getApp().given().get(API_ROOT + "/" + randomNumeric(4));
+        final Response response = getApp().given().get(API_ROOT + "/" + insecure().nextNumeric(4));
```

Comment:
Java 17 has `RandomGenerator`, that would be my preference. Anyway, my question:  I am not sure why is using one `org.apache.commons.lang3.RandomStringUtils` method over another method better. Is that because the original method was deprecated?

Reply:
methods for RandomStringUtils `randomNumber` and `randomAlphabetic` are deprecated. Construct I use is one of the recommended replacements. I used insecure because we don't need cryptographically secured randomness here.